src/themes/wmc/theme.css

```diff
@@ -138,178 +138,178 @@ html {
     color: inherit;
-    opacity: .5
+    opacity: 0.5;
 }
 
 .toast {
-    background: #081B3B;
+    background: #081b3b;
     color: #fff;
-    color: rgba(255, 255, 255, .87)
+    color: rgba(255, 255, 255, 0.87);
 }
 
 .appfooter,
 .formDialogFooter:not(.formDialogFooter-clear) {
-    background: #0C2450;
-    background: linear-gradient(to top, #0C2450, #081B3B);
-    color: rgba(255, 255, 255, .78)
+    background: #0c2450;
+    background: linear-gradient(to top, #0c2450, #081b3b);
+    color: rgba(255, 255, 255, 0.78);
 }
 
 .itemSelectionPanel {
-    border: 1px solid #00a4dc
+    border: 1px solid #00a4dc;
 }
 
 .selectionCommandsPanel {
     background: #00a4dc;
-    color: #fff
+    color: #fff;
 }
 
 .upNextDialog-countdownText {
-    color: #00a4dc
+    color: #00a4dc;
 }
 
 .alphaPickerButton {
     color: #999;
-    color: rgba(255, 255, 255, .5);
-    background-color: transparent
+    color: rgba(255, 255, 255, 0.5);
+    background-color: transparent;
 }
 
 .alphaPickerButton-selected,
 .alphaPickerButton-tv:focus {
     background-color: #00a4dc;
-    color: #fff !important
+    color: #fff !important;
 }
 
 .detailTableBodyRow-shaded:nth-child(even) {
     background: #1c1c1c;
-    background: rgba(0, 0, 0, .3)
+    background: rgba(0, 0, 0, 0.3);
 }
 
 .listItem-border {
-    border-color: rgba(0, 0, 0, .3) !important
+    border-color: rgba(0, 0, 0, 0.3) !important;
 }
 
 .listItem:focus {
-    background: #333
+    background: #333;
 }
 
 .progressring-spiner {
-    border-color: #00a4dc
+    border-color: #00a4dc;
 }
 
 .button-flat-accent,
 .button-link {
-    color: #00a4dc
+    color: #00a4dc;
 }
 
 .mediaInfoText {
     color: #ddd;
-    background: rgba(170, 170, 190, .2)
+    background: rgba(170, 170, 190, 0.2);
 }
 
 .mediaInfoTimerIcon,
 .starIcon {
-    color: #CB272A
+    color: #cb272a;
 }
 
 .emby-input,
 .emby-textarea {
     color: inherit;
-    background: rgba(255, 255, 255, .2);
-    border: .07em solid rgba(255, 255, 255, .135);
-    border-radius: .15em
+    background: rgba(255, 255, 255, 0.2);
+    border: 0.07em solid rgba(255, 255, 255, 0.135);
+    border-radius: 0.15em;
 }
 
 .emby-input:focus,
 .emby-textarea:focus {
-    border-color: #00a4dc
+    border-color: #00a4dc;
 }
 
 .emby-select-withcolor {
     color: inherit;
-    background: rgba(255, 255, 255, .2);
-    border: .07em solid rgba(255, 255, 255, .135)
+    background: rgba(255, 255, 255, 0.2);
+    border: 0.07em solid rgba(255, 255, 255, 0.135);
 }
 
-.emby-checkbox:checked+span+.checkboxOutline,
+.emby-checkbox:checked + span + .checkboxOutline,
 .emby-select-withcolor:focus {
-    border-color: #00a4dc
+    border-color: #00a4dc;
 }
 
-.emby-checkbox:focus+span+.checkboxOutline {
+.emby-checkbox:focus + span + .checkboxOutline {
     border-color: #fff;
 }
 
-.emby-checkbox:focus:not(:checked)+span+.checkboxOutline {
+.emby-checkbox:checked + span + .checkboxOutline,
+.itemProgressBarForeground {
+    background-color: #00a4dc;
```

Comment:
why no `color: #fff` moved?

Reply:
Git is getting confused. I'm moving stuff around the file. The color #fff is further down.